<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background: rgb(255, 255, 255)</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLabel" name="tetraFileLabel">
      <property name="text">
       <string>Tetra File:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="logo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>172</width>
        <height>152</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>172</width>
        <height>152</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QPlainTextEdit" name="input"/>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="outputLabel">
      <property name="text">
       <string>Output:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QTextEdit" name="output">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="5" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="enterButton">
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: #333;
border: 2px solid #555;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888);
min-width: 75px;
}

QPushButton:hover {
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb);
}

QPushButton:pressed {
background: qradialgradient(cx: 0.4, cy: -0.1,
fx: 0.4, fy: -0.1,
radius: 1.35, stop: 0 #fff, stop: 1 #ddd);
}</string>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Recent"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Tetra"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>Build</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="separator"/>
    <addaction name="actionBuild"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionDebug"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBuild"/>
   <addaction name="menuDebug"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionPrint"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionBuild"/>
   <addaction name="actionRun"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionDebug"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/icons/Tetra Resources/icons/appbar.save.png</normaloff>:/icons/Tetra Resources/icons/appbar.save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/icons/Tetra Resources/icons/appbar.printer.text.png</normaloff>:/icons/Tetra Resources/icons/appbar.printer.text.png</iconset>
   </property>
   <property name="text">
    <string>Print...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_Tetra">
   <property name="text">
    <string>About Tetra</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/icons/Tetra Resources/icons/appbar.undo.png</normaloff>:/icons/Tetra Resources/icons/appbar.undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/icons/Tetra Resources/icons/appbar.redo.png</normaloff>:/icons/Tetra Resources/icons/appbar.redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/icons/Tetra Resources/icons/appbar.control.play.png</normaloff>:/icons/Tetra Resources/icons/appbar.control.play.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDebug">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/icons/Tetra Resources/icons/appbar.bug.png</normaloff>:/icons/Tetra Resources/icons/appbar.bug.png</iconset>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionOpen_Recent">
   <property name="text">
    <string>Open Recent</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/icons/Tetra Resources/icons/appbar.protractor.png</normaloff>:/icons/Tetra Resources/icons/appbar.protractor.png</iconset>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="AppResources.qrc">
     <normaloff>:/icons/Tetra Resources/icons/appbar.control.stop.png</normaloff>:/icons/Tetra Resources/icons/appbar.control.stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AppResources.qrc"/>
 </resources>
 <connections/>
</ui>
